node/npm/glide/index.ts

```diff
@@ -185,19 +185,18 @@ function initialize() {
         SignedEncoding,
         UnsignedEncoding,
         UpdateByScore,
-        createLeakedArray,
-        createLeakedAttribute,
-        createLeakedBigint,
-        createLeakedDouble,
-        createLeakedMap,
-        createLeakedString,
-        parseInfoResponse,
         Script,
         ObjectType,
         ClusterScanCursor,
         BaseClientConfiguration,
         GlideClusterClientConfiguration,
         LevelOptions,
+        ReturnTypeRecord,
+        ReturnTypeMap,
+        WritePromiseOptions,
```

Comment:
We should not export `WritePromiseOptions`

Reply:
addressed

Reply:
Agreed.  It's marked as `@internal`.